### ISSUE TITLE
Accessibility/Groups/Members Tab: Serious a11y violations

### ISSUE THREAD

**Issue by agagancarczyk** [keycloak/keycloak#19319]
### Before reporting an issue

- [X] I have searched existing issues
- [X] I have reproduced the issue with the latest release

### Area

admin/ui

### Describe the bug

Two serious violations were detected.

![Screenshot 2023-03-24 at 16 27 16](https://user-images.githubusercontent.com/4890675/227585667-d2e9a336-4fd9-4746-ab78-9c4cc3b23cf5.png)

![Screenshot 2023-03-24 at 16 27 23](https://user-images.githubusercontent.com/4890675/227585686-c0f289e0-08f3-4981-a3a3-42e4389167b5.png)

### Version

21.0.0

### Expected behavior

Violations are fixed.

### Actual behavior

Violations are not fixed.

### How to Reproduce?

Go to group details ->  members tab -> add a member -> scan the page.

Use the deque axe tool.
